### PR TITLE
[TASK] Upgrade DDEV

### PR DESCRIPTION
**Pull request by lukaszuznanski**

### CODE REVIEW

File: .ddev/config.yaml [hunk 1]
```diff
@@ -1,33 +1,31 @@
 name: tea
 type: php
 docroot: ""
-php_version: "8.1"
+php_version: "8.3"
```
Comment (oliverklee) [inline .ddev/config.yaml:4]: Please let's move the PHP upgrade to a separate PR. (Also, we need to stay below PHP 8.3 in order to not break PHP-CS-Fixer at this moment.)

File: .ddev/config.yaml [hunk 2]
```diff
@@ -1,33 +1,31 @@
...
-mutagen_enabled: false
+    type: mariadb
+    version: "10.5"
 hooks:
-  post-start:
-  - exec: composer install --no-progress
-omit_containers: [dba, ddev-ssh-agent]
+    post-start:
+        - exec: composer install --no-progress
+omit_containers: [ddev-ssh-agent]
```
Comment (oliverklee) [inline .ddev/config.yaml:17]: Shouldn't we still omit the DBA container? Or is this related to the DDEV upgrade?

File: .ddev/config.yaml [hunk 3]
```diff
@@ -1,33 +1,31 @@
...
 xdebug_enabled: false
 additional_hostnames: []
 additional_fqdns: []
 database:
-  type: mariadb
-  version: "10.5"
-nfs_mount_enabled: false
-mutagen_enabled: false
+    type: mariadb
+    version: "10.5"
```
Comment (oliverklee) [inline .ddev/config.yaml:13]: I'd prefer to have the formatting cleanup in a separate PR as a pre-change in order to separate the formatting changes from the real configuration changes.

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: .ddev/config.yaml (hunk 3 above)
Comment (lukaszuznanski) [reply .ddev/config.yaml:13]: Those changes comes from ddev upgrade, it automatically reformat this file

File: .ddev/config.yaml (hunk 2 above)
Comment (lukaszuznanski) [reply .ddev/config.yaml:17]: Will fix, related to ddev upgrade, but it should be migrated also.

File: .ddev/config.yaml (hunk 1 above)
Comment (lukaszuznanski) [reply .ddev/config.yaml:4]: Ok

**Review by oliverklee — APPROVED** [review]
